calblueprint/ogsc
# User Profile Styling Updates

# User Profile Styling Updates

- Update styling of user profile to reflect new design
- found and fixed a bug with editing admin profiles (wasn't working on main)

## TODO

- not sure what to do with top edit button, but just asked Rachel for a design!

## How to Test/Use PR feature

go to any user profile and make sure it matches the following design and works as intended

![image](https://user-images.githubusercontent.com/65790318/111078604-8d536f80-84b3-11eb-8dd8-3e173a135017.png)
![image](https://user-images.githubusercontent.com/65790318/111078622-9e9c7c00-84b3-11eb-9544-38f2bd96b3b4.png)


## PR Checklist

Does your branch (check if true):
[X] work when you run `yarn build`?
[X] successfully run `yarn:db-migrate up` without yielding any errors?
[X] successfully run `yarn prisma introspect` without yielding any errors?
[X] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Screenshots

_Required for visual changes_
![image](https://user-images.githubusercontent.com/65790318/111078661-e0c5bd80-84b3-11eb-92f0-c5a5d8c53881.png)
![image](https://user-images.githubusercontent.com/65790318/111078674-ef13d980-84b3-11eb-814f-e666ffe6e5a7.png)


CC: @sydbui
